### DISCUSSION TITLE
WAV FILES

### DISCUSSION THREAD

**Discussion by WOODY1958UK** [EdgeTX/edgetx#3375] [Ideas]
HI - FIRST PLEASE E4XCUSE CAPS BUT I AM PARTIALLY SIGHTED AND IT HELPS - SO AN ISSUE MY FLYING BUDDIES AND I HAVE HAD IS WAV FILES FOR - OPEN TX - EDGE TX - ETHOS - OTHER RADIOS AND MANUFACTURERS - SO I HAVE DECIDED TO DO SOMETHING ABOUT IT - SO NOW HAVE A WEBSITE WITH ONE AIM TO PROVIDE A ONE STOP SOLUTION TO WAV FILES FOR RC MODELS - BEFORE I LAUNCH IT - COULD SARA - RYAN - LIBBY - MICHELE - GUY PLEASE CONTACT ME SO I CAN GET THERE PERMISSION TO POST THEIR WAV FILES ON THE SITE - SECOND - I HAVE THE ABILITY TI WRITE NEARLY ANY COMMAND THAT'S NOT YET PROVIDED - THIRD - I THINK I CAN TRANSLATE THESE COMMANDS INTO OVER 30 LANGUAGES AND DIALECTS - I WILL ALSO TRY TO SET UP A DONATION PAGE SO I CAN PASS FUNDS BACK TO THE ORIGINAL TEAMS - AND TO PAY FOR SOFTWARE AND WEBSITE - THOUGHTS PLEASE

**Comment by pfeerick** [comment]
> BEFORE I LAUNCH IT - COULD SARA - RYAN - LIBBY - MICHELE - GUY PLEASE CONTACT ME SO I CAN GET THERE PERMISSION TO POST THEIR WAV FILES ON THE SITE

THESE ARE ALL MICROSOFT AZURE TEXT TO SPEECH VOICES, SO SHOULD BE ABLE TO BE FREELY DISTRIBUTED - CHECK THE GUIDE HERE https://github.com/EdgeTX/edgetx-sdcard-sounds#voices

IF THERE ARE FURTHER PHRASES WANTED IN THE STOCK SD CARD PACKS, PEOPLE ARE WELCOME TO MAKE SUGGESTIONS AND MAKE PULL REQUESTS TO IMPROVE AND ADD TO THE EXISTING PACKS. BUT IT WOULD BE NICE TO HAVE A ONE-STOP SHOP FOR SOUND FILES FOR MULTIPLE SYSTEMS - LIKE THE SKYRACOON SITE FOR MODEL IMAGES :)

**Comment by WOODY1958UK** [comment]
HI OKAY GREAT WEBSITE SOON WILL CONTACT YOU AGAIN WHEN R & R THANKS CATCH YOU LATER TAKE CARE DAVID - WOODY1958UK
